### DISCUSSION TITLE
Input field covered by calendar picker on first click.

### DISCUSSION THREAD

**Discussion by protricker** [easepick/easepick#29] [Q&A] [answered]
Hello. Everything works fine. But i've noticed one strange thing. 

On first click calendar top padding is not correct, it covers input field.
![image](https://user-images.githubusercontent.com/62597533/161987963-ce0bc842-94eb-4020-a7a9-2cfd40d51ffb.png)
After second click and subsequents everything is ok. 
![image](https://user-images.githubusercontent.com/62597533/161988168-79e4ccfb-5f7a-4d1d-98b4-2ccbf7c00921.png)
Is there any workaround without css tricks?

**Comment by wakirin** [comment, marked as answer]
Hello,
I saw this bug, thanks for reminding me. 
Currently I can suggest doing something like this: https://jsfiddle.net/waki/04ohq52z/
I will investigate this bug and fix it in the next release.

**Reply by protricker** [reply]
Thank you for answer and thank you for for awesome product!
